### PR TITLE
Update metadata.yaml for Ashwin Skelly

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Commits were made by @anskelly, I just opened the PR -- but that means I can't approve

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
The diff looks fine, but I'm not sure what to make of the "this branch is out-of-date with the base branch" warning.

File: content/metadata.yaml [hunk 1]
```diff
@@ -434,11 +434,11 @@ authors:
     affiliations:
       - Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
       - Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
-    code of conduct:
+    code of conduct: yes
```
Comment (rando2) [inline content/metadata.yaml:437]:
```suggestion
    code of conduct:
```

Comment (rando2) [reply content/metadata.yaml:437]: @anskelly I'm not sure if I did this (I am a little out of it this week) but it is causing a bit of an issue, would you be able to just accept this change? Sorry about this!!!

Comment (anskelly) [reply content/metadata.yaml]: @rando2 Yup no worries! I think I just accepted it! <3

**Review by agitter — APPROVED** [review]

**Comment by rando2** [comment]
These changes were merged in by 8a7d189 (I followed the github command line instructions so that I could rebase to avoid any issues with out-of-date branches, and it seems like it's not registering as part of this PR)

**Pull request closed without merging**